app/views/spree/admin/payments/new.html.haml

```diff
@@ -1,4 +1,4 @@
-= render partial: 'spree/admin/shared/order_tabs', locals: { current: 'Payments' }
```

Comment:
I still fail to see where this has gone. Maybe you haven't pushed the last commit?